examples/benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2016, Dennis Sitelew
+cmake_minimum_required(VERSION 3.0)
+
+if(NOT CMAKE_BUILD_TYPE)
+  message(WARNING "You shouldn't benchmark a debug build")
```

Comment:
@yowidin 
Sometimes you'd like to debug the benchmarking code itself. I think that devs are responsible enough to use Release version after they have already made an effort to use google/benchmark.